src/interface/substrate_query.js

```diff
@@ -395,6 +416,16 @@ module.exports = {
                 } catch (e) {
                     return {'error': e.toString()};
                 }
+	    case 'staking/bonded':
+                try {
+		    if (!param2) {
+                        return {'error': 'You did not enter the stash '
+                                + 'address that needs to be queried'};
+                    }
```

Comment:
Can you make sure this curly bracket and the subsequent return statement are aligned with the `if` above